### ISSUE TITLE
Building website on Windows fails with error about VS2010

### ISSUE THREAD

**Issue by jonathandturner** [rust-lang/www.rust-lang.org#719]
# Technical Bug Report
## Your Environment
Windows 10 Home 1809
Visual Studio Community 2017

## Is this an issue with Rust, CSS, or HTML (or a combination)?
Seems to be something trying to build a C++ component?

## Expected Behavior
"cargo watch -x run" should work with an operational Rust setup on Windows

## Current Behavior
I'm currently seeing the website fail to build on Windows with this error:

```
  Building Static LibSass
C:\Program Files (x86)\Microsoft Visual Studio\2017\Community\Common7\IDE\VC\VCTargets\Microsoft.Cpp.Platform.targets(65,5): error MSB8020: The build tools for Visual Studio 2010 (Platform Toolset = 'v100') cannot be found. To build using the v100 build tools, please install Visual Studio 2010 build tools.  Alternatively, you may upgrade to the current Visual Studio tools by selecting the Project menu or right-click the solution, and then selecting "Retarget solution". [C:\Source\thirdparty\www.rust-lang.org\target\debug\build\sass-sys-1ea3913db2b938d5\out\build\win\libsass.vcxproj]
Done Building Project "C:\Source\thirdparty\www.rust-lang.org\target\debug\build\sass-sys-1ea3913db2b938d5\out\build\win\libsass.vcxproj" (default targets) -- FAILED.
Done Building Project "C:\Source\thirdparty\www.rust-lang.org\target\debug\build\sass-sys-1ea3913db2b938d5\out\build\win\libsass.sln" (default targets) -- FAILED.

Build FAILED.

"C:\Source\thirdparty\www.rust-lang.org\target\debug\build\sass-sys-1ea3913db2b938d5\out\build\win\libsass.sln" (default target) (1) ->
"C:\Source\thirdparty\www.rust-lang.org\target\debug\build\sass-sys-1ea3913db2b938d5\out\build\win\libsass.vcxproj" (default target) (2) ->
(PlatformPrepareForBuild target) ->
  C:\Program Files (x86)\Microsoft Visual Studio\2017\Community\Common7\IDE\VC\VCTargets\Microsoft.Cpp.Platform.targets(65,5): error MSB8020: The build tools for Visual Studio 2010 (Platform Toolset = 'v100') cannot be found. To build using the v100 build tools, please install Visual Studio 2010 build tools.  Alternatively, you may upgrade to the current Visual Studio tools by selecting the Project menu or right-click the solution, and then selecting "Retarget solution". [C:\Source\thirdparty\www.rust-lang.org\target\debug\build\sass-sys-1ea3913db2b938d5\out\build\win\libsass.vcxproj]

    0 Warning(s)
    1 Error(s)

Time Elapsed 00:00:00.55
```

## Step to Reproduce
At least on my machine (with VS2017 Community installed, and a working Rust), all I had to do was check out the latest website, have cargo-watch installed, and run 'cargo watch -x run'.

**Comment by yoshuawuyts** [comment]
Dang, yeah it seems our SASS engine if failing to build. We're halfway through replacing it with Tachyons, but for the time being it is what holds all of our styling together.

I'm not sure what might be causing this, and don't have access to a Window machine. Perhaps asking on the libsass repo might be helpful? https://github.com/sass/libsass

Sorry this is happening!

**Comment by skade** [comment]
Blocked on the removal of skeleton towards Tachyon https://github.com/rust-lang/www.rust-lang.org/issues/183

**Comment by XAMPPRocky** [comment]
Thank you for your issue! I'm going to close this, as whether this is still an issue is unclear and there's been no traffic, if there still are problems building the website on windows. Please file a new issue.
